src/core/communication.hpp

```diff
@@ -36,7 +36,7 @@
  *  the action issued. If applicable, the second parameter is the node
  *  number of the slave this request is dedicated to.
  *
- *  To add new actions (e.g. to implement new Tcl commands), do the
+ *  To add new actions (e.g. to implement new Python features), do the
```

Comment:
this does not depend on the script language to reach the interface in the core. just call it ```interface functionality``` or similar.